casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala

```diff
@@ -69,7 +73,7 @@ class InterpreterUtilTest
               runtimeManager,
               BlockData(
                 now,
-                0,
+                seqNum,
```

Comment:
I'm unsure if it is okay to equate sequence numbers with block numbers.
@itegulov can you verify that this is okay?

Reply:
More general question: how to get last block number required for deploy as `validAfterBlockNumber`? https://github.com/rchain/rchain/pull/2811#pullrequestreview-323624096

Reply:
I think it's OK for tests because of: 
```scala
      nextId           = if (block.seqNum == 0) currentId + 1L else block.seqNum.toLong
      newPostState     = block.body.state.copy(blockNumber = nextId)
```
in the `IndexedBlockDagStorage` (thanks for pointing out to it @tymm )